cmd/neofs-cli/modules/object.go

```diff
@@ -427,6 +432,11 @@ func getObjectHash(cmd *cobra.Command, _ []string) error {
 		return err
 	}
 
+	salt, err := hex.DecodeString(cmd.Flag(getRangeHashSaltFlag).Value.String())
```

Comment:
It would be nice to trim `0x` prefix from salt value just in case.

Reply:
Done